modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/js/components/FileEntryPicker.tsx

```diff
@@ -0,0 +1,351 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayAlert, {DisplayType} from '@clayui/alert';
+import ClayButton, {ClayButtonWithIcon} from '@clayui/button';
+import {Text} from '@clayui/core';
+import {ClayInput} from '@clayui/form';
+import ClayLoadingIndicator from '@clayui/loading-indicator';
+import PropTypes from 'prop-types';
+import React, {useEffect, useRef, useState} from 'react';
+
+const MALFORMED_JSON = '{]/';
```

Comment:
This was to prevent sending an invalid FTD or file to be validated on the CX side. But [this approach](https://github.com/liferay-platform-experience/liferay-portal/pull/460#discussion_r1473051419) makes this unnecessary